ansible/roles/hotpot-installer/tasks/main.yml

```diff
@@ -68,6 +68,12 @@
     dest: "{{ opensds_config_dir }}"
   when: deploy_project != "gelato"
 
+- name: generate certificates
+  shell: "{{ item }}"
+  with_items:
+    - bash ./script/create_cert.sh
+  become: yes	  
```

Comment:
You missed this line: `when: deploy_project != "gelato"`

Reply:
I added the line. Please review again.